libs/platform/table/table.component.ts

```diff
@@ -488,6 +492,10 @@ export class TableComponent<T = any>
     @Input()
     minimumColumnWidth = TABLE_COLUMN_MIN_WIDTH;
 
+    /** Whether to display the table footer. */
+    @Input()
+    showTableFooter = false;
```

Comment:
As far as I remember, we decided some time ago to use `booleanAttribute` in such cases. 
```
 @Input({ transform: booleanAttribute })
 showTableFooter = false;
```

Then the user can do 
```
<fdp-table [dataSource]="source" [trackBy]="trackBy" emptyTableMessage="No data found" showTableFooter>
```

instead of
```
<fdp-table [dataSource]="source" [trackBy]="trackBy" emptyTableMessage="No data found" [showTableFooter]="true">
```